src/lichess.js

```diff
@@ -75,7 +75,7 @@ var makeRequest = (function() {
     }
     processRequest();
     return makeRequest;
-})();
+}());
```

Comment:
Ok.  I don't have a stylistic preference either. Let's leave it the default for eslint. Easier that way.